platform-includes/sourcemaps/legacy-uploading-methods/javascript.mdx

```diff
@@ -280,6 +280,12 @@ To troubleshoot your source maps set up, you can either:
 
 To use the automated verification process, install and configure the Sentry [Command Line Interface](/product/cli/). Then, use the `sourcemaps explain` command, calling it with the relevant event ID, found in the top-left corner of the **Issue Details** page in [sentry.io](https://sentry.io).
 
+<Alert title="Note" level="warning">
+
+The `sourcemaps explain` command requires an auth token with the following scopes: `project:admin`, `release:admin`, `event:read`, `organisation:read`. We recomend creating an [internal integration token](https://docs.sentry.io/product/integrations/integration-platform/internal-integration/) over a personal token.  
```

Comment:
```suggestion
The `sourcemaps explain` command requires an auth token with the following scopes: `project:admin`, `release:admin`, `event:read`, `organisation:read`. We recomend creating an [internal integration token](/product/integrations/integration-platform/internal-integration/) over a personal token.  
```

Reply:
Updated your suggestion to fix some typos